antora-playbook-dev.yml

```diff
@@ -1,6 +1,6 @@
 site:
   title: TinyMCE Documentation (Dev)
-  url: https://tiny.cloud/docs
+  url: https://www.tiny.cloud/docs
```

Comment:
should this be? 

https://www.tiny.cloud/docs/tinymce/6/

Reply:
nvm i see why this is like that {{site.url}} too much too change now

Reply:
Yeah, this is meant to be the root of the docs site that contains all products/versions.